src/components/form/Label.tsx

```diff
@@ -3,6 +3,13 @@ export function Label({
   required,
   ...props
 }: React.LabelHTMLAttributes<HTMLLabelElement> & {
+  /**
+   * UI depending on the value:
+   *
+   * - `true` - a red '*' (star) next to the label
+   * - `false` - text '(optional)' next to the label
+   * - `undefined` - nothing appears next to the label
+   */
   required?: boolean;
```

Comment:
Found it confusing that UI was different for `required` values `false` and `undefined`, so added this comment for future clarity.